DataDog/storedog
# WEB-3381 | backend setup for DBM

## Description
Created a new backend service with a bad query to work with DBM in the application

## How to test

1. Pull this branch down locally
2. Run `cp .env.template .env && cp .env.template ./deploy/docker-compose/.env && cp .env.template ./services/frontend/site/.env.local`
3. run `docker-compose --profile dbm up`
4. goto `http://localhost:7595/get-item` in your browser or run a `GET` in your favorite API testing suite (postman, insomnia, etc)
5. Ensure you get back an item, it should look like the following 
```json
{
  "description": "Drive admit newspaper tax war company partner.",
  "id": 8903,
  "image_url": "https://placekitten.com/329/165",
  "last_hour": "7",
  "order_count": "4985"
}
```

That is it, I need to create the frontend connection for this, but wanted to get some eyes on this. Also it looks like the real struggle is getting DBM setup with our way of doing things. I may need to try a few things, following the docs, I'm getting snagged with the setting of the different values for the config and the portion where we run the setting of `shared_preload_libraries = pg_stat_statements`. For whatever reason it just fails every time, need to do more digging but the groundwork is here.
